PubSub/src/BatchPublisher.php

```diff
@@ -82,7 +82,8 @@ public function __construct($topicName, array $options = [])
      * ]);
      * ```
      *
-     * @param array $message [Message Format](https://cloud.google.com/pubsub/docs/reference/rest/v1/PubsubMessage).
+     * @param array $message [Message Format]
```

Comment:
The documentation generator requires links be on a single line. If the line length is longer than 120 chars, use `@codingStandardsIgnoreStart/End` to prevent CI failures, e.g.:

```
     * @codingStandardsIgnoreStart
     * @param array $message [Message Format](https://cloud.google.com/pubsub/docs/reference/rpc/google.pubsub.v1#pubsubmessage).
     * @codingStandardsIgnoreEnd
```